### DISCUSSION TITLE
Gas motor in INAV Fixed Wing

### DISCUSSION THREAD

**Discussion by mickey2011** [iNavFlight/inav#10078] [Support]
I would appreciate some help/links etc for a Gas motor in INAV 7.1.1 Fixed Wing setup, especially how to setup/control the servo activated throttle. Stabilized Throttle? or  RC Throttle? and what pins to plug a throttle servo into, now that the outputs can be assigned to different timers etc. Thanks

**Comment by sensei-hacker** [comment]
For the most part it's the same as an ESC controlling an electric motor. It's just a servo rather than an ESC.  Set stabilized throttle, idle throttle, etc all the same. It can be plugged into any pins that are assigned as a servo.

Vibration is the big difference. The vibration from a gas or nitro engine will cause problems with the IMU unless care is taken.

You may want to jump on the INAV Discord and speak to someone named JBD. They've done this, and may have taken some notes. Perhaps they would be willing to add their notes to the wiki.

You will want to use the latest INAV release - there was a bug in an earlier version which affected throttle servos.

Gas _is_ unusual, so please be extra careful to follow the standard safety procedures. It's always possible that in some corner case there's a behavior that has ESCs in mind. Prop off if you're messing with the plane on the bench and don't intend to fly, keep fingers away from the prop, etc.  Standard stuff, but just a little more important whenever you're working with something other than the typical craft a million other people fly.

**Reply by mickey2011** [reply]
Thanks for your reply, but I'm no further ahead. Tried to contact JBD but no answer. If we forget about "Gas motor" for now, I'm struggling to understand what the 'Timer reassignment' in vs 7 does. A FC that I am experimenting with is a 'WINGFC' it comes with two Motor outputs M1 and M2 and 4 servo outputs S3-S6. Now if I make all the TIMERS 'Servos' does that mean I now have 6 servo outputs and can drive 6 servos? If so do the additional servos connect to M1 and M2? but where are these two servos configured? Any help appreciated, thanks

**Reply by sensei-hacker** [reply]
Yes. You have six outputs, S1 through S6.
You can decide how many motors and how many servos you want.

Each timer can run either servos or motors on the outputs attached to that timer. 

All servos are configured in the servo mixer, on the mixer page.

**Comment by mickey2011** [comment]
Hello Sensi, thanks for your help again, but still no go, well not on this
FC anyway. But I can't get any indicated outputs even on INAV Configurator.

I cannot find anything about Timer assignment on INAV Github, it's a hard
site to navigate, very frustrating.

Cheers

**Comment by sensei-hacker** [comment]
You can completely ignore timers are just click the button to add servos.

The default for timers on INAV is "auto", meaning it will automatically assign timers to whatever servos or motors you add.

If there are motors (ESCs) listed where you don't want them, click the delete button on those before adding.
